### DISCUSSION TITLE
*BSD Support?

### DISCUSSION THREAD

**Discussion by anoduck** [VSCodium/vscodium#1820] [Ideas]
Sometimes it feels like suggesting support for the BSDs is an "unpopular opinion". Regardless, this would be nice to have.

**Comment by daiyam** [comment]
Which BSD you would like? For the REH or the classic (IDE/UI) version?

**Reply by anoduck** [reply]
@daiyam Classic, most definitely the classic. 

REH is pretty ding dang doodley slick, and probably would be easier to port over, but it is already available for the platform using remote hosted services like gitpod. There is definitely, not as many IDE options for OpenBSD as there are for Linux, it would be nice to have VSCodium among the list.

**Comment by kevinlearnscoding** [comment]
I personally favor FreeBSD and OpenBSD. Of the two, I'd push for FreeBSD first.

**Reply by daiyam** [reply]
VSCode is already available with https://github.com/tagattie/FreeBSD-VSCode/tree/master. So it could be done.
